README.md

```diff
@@ -13,6 +13,6 @@ proper docs
 ## iContact Module
 
 - install the files in the icontact-package as per the instructions in our documentation on creating
-an iContact Plugin. 
+an iContact Plugin. https://documentation.cpanel.net/display/CKB/How+To+Create+Custom+Server+Notifications
```

Comment:
You probably should have used markdown to add the link:
[cPanel's Documentation](https://documentation.cpanel.net/display/CKB/How+To+Create+Custom+Server+Notifications "How to Create Custom Server Notifications")